digos-ambassador/Content/Transformations/Species/snake/Arm.yml

```diff
@@ -0,0 +1,9 @@
+part: Arm
+species: snake
+description: "Transforms the target's {@side} arm into that of an anthro Serpent.”
+default_base_colour: "green"
+is_nsfw: false
+shift_message: "Smooth {@colour} scales which mildly shimmer spread over {@target}'s {@side} arm. {@f|Their} newly scaled flesh ripples as cords of potent fibrous muscle coil and stitch themselves into shape beneath their skin. The bones in their {@side} hand twist and pop as they stretch out and grow slightly longer while small inhuman claws push out from their fingertips.”
+grow_message: "{@target}'s {@side} shoulder pops and shimmers, erupting outward into an arm. It emerges covered in smooth {@Colour} scales which shimmer as they reveal every coiled fibrous bundle of potent muscle. {@f|Their} long slender fingers are tipped with with small inhuman claws assure nothing will escape their grasp.”
+single_description: "{@f|They have} a snake-like {@side} arm, with shimmering smooth scales tightly wrapped over potent cords of fibrous muscle. {@f|Their} long slender fingers are tipped with with small inhuman claws assure nothing will escape their grasp.”
```

Comment:
"{@f|Their} long slender fingers are tipped with with small inhuman claws assure nothing will escape their grasp."

Missing word? Maybe add "that" or "which" between "claws" and "assure".